testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/TestHiveTransactionalTable.java

```diff
@@ -1652,6 +1652,26 @@ public void testFilesForAbortedTransactionsIgnored()
         }
     }
 
+    @Test(groups = HIVE_TRANSACTIONAL)
+    public void testDoubleUpdateAndThenReadFromHive()
+    {
+        onHive().executeQuery("DROP TABLE IF EXISTS test_double_update");
+        onTrino().executeQuery(
+                "CREATE TABLE test_double_update ( " +
+                        "column1 INT, " +
+                        "column2 VARCHAR " +
+                        ") " +
+                        "WITH ( " +
+                        "   transactional = true " +
+                        ");");
+        onTrino().executeQuery("INSERT INTO test_double_update VALUES(1, 'x');");
+        onTrino().executeQuery("INSERT INTO test_double_update VALUES(2, 'y');");
+        onTrino().executeQuery("UPDATE test_double_update SET column2 = 'xy1';");
+        onTrino().executeQuery("UPDATE test_double_update SET column2 = 'xy2';");
+        assertThat(onHive().executeQuery("SELECT * FROM test_double_update;")).containsOnly(row(1, "xy2"), row(2, "xy2"));
+        onHive().executeQuery("DROP TABLE IF EXISTS test_double_update");
```

Comment:
drop `;` from query strings

Reply:
ok

Reply:
Can we check if it reads correctly in both Trino and Hive ?

Reply:
sure, will do